src/source-af-packet.c

```diff
@@ -1736,11 +1736,12 @@ static int AFPSetupRing(AFPThreadVars *ptv, char *devname)
     }
 
 #ifdef HAVE_HW_TIMESTAMPING
-    int req = SOF_TIMESTAMPING_RAW_HARDWARE;
-    if (setsockopt(ptv->socket, SOL_PACKET, PACKET_TIMESTAMP, (void *) &req,
-                sizeof(req)) < 0) {
-        SCLogWarning("%s: failed to activate hardware timestamping on packet socket: %s", devname,
-                strerror(errno));
+    if ((ptv->flags & AFP_DISABLE_HWTIMESTAMP) == 0) {
```

Comment:
I almost want an SCLogConfig here telling me  hardware timestamping is being disabled.